src/app/views/developer/developer.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+
+  <app-library-top-level-menu [menuItems]="topLevelMenuItems"></app-library-top-level-menu>
+
+  <ng-container *ngIf="libraryItemsLoaded else spinner">
+    <div class="row">
+      <div class="col-md-4 col-lg-3 col-xxl-3">
+        <app-library-second-level-menu [menuItems]="secondLevelMenuItems"></app-library-second-level-menu>
```

Comment:
Missing css class `app-library-second-level-menu`. This causes the left menu not to be sticky.

Reply:
Other than that, this looks really good. The structure looks really good, and the demo app will be much better to work with.